src/main/java/co/cask/gcp/bigquery/BigQuerySink.java

```diff
@@ -189,6 +196,8 @@ public String getOutputFormatClassName() {
                                                            .collect(Collectors.toList()));
       context.record(Collections.singletonList(operation));
     }
+
+    validateSchema(config);
```

Comment:
this validation should as early as possible.

Reply:
@albertshau Can this validation also be done in configurePipeline so that we fail at deployment? Is that possible?

Reply:
We generally do not make calls with clients at configure time and only validate static configuration. The reason is that we don't want to stop people from deploying pipelines before all their dependencies are setup.

Reply:
👍